selenium_tests/course_info/course_info_base_test_case.py

```diff
@@ -46,52 +58,28 @@ def setUp(self):
         self.course_info_parent_page.select_course_info_link()
 
         # check if page is loaded (which will also set the focus on the tool)
-        self._search_page = CourseSearchPageObject(self.driver)
-        self.assertTrue(self._search_page.is_loaded())
+        self.assertTrue(self.search_page.is_loaded())
 
-    def search_for_course(self, type=None, school=None, term=None, year=None,
+    @classmethod
+    def search_for_course(cls, type=None, school=None, term=None, year=None,
                           search_term=''):
         if type:
-            self._search_page.select_course_type(type)
+            cls.search_page.select_course_type(type)
         if school:
-            self._search_page.select_school(school)
+            cls.search_page.select_school(school)
         if term:
-            self._search_page.select_term(term)
+            cls.search_page.select_term(term)
         if year:
-            self._search_page.select_year(year)
-        self._search_page.submit_search(search_term)
-
-    def _api_post(self, path=None, body=None):
-        request_args, url = self._api_prep_request(path=path, body=body)
-        return requests.post(url, **request_args)
-
-    def _api_get(self, path=None, params=None):
-        request_args, url = self._api_prep_request(path=path, params=params)
-        return requests.get(url, **request_args)
-
-    def _api_delete(self, path=None, body=None):
-        request_args, url = self._api_prep_request(path=path, body=body)
-        return requests.delete(url, **request_args)
-
-    def _api_prep_request(self, body=None, params=None, path=None):
-        params_plus_defaults = (params or {}).copy()
-        params_plus_defaults.update({'format': 'json'})
-
-        request_args = {
-            'data': body,
-            'headers': {
-                'Authorization': "Token {}".format(
-                    settings.ICOMMONS_REST_API_TOKEN)
-            },
-            'params': params_plus_defaults
-        }
-
-        url = "{}/{}/{}".format(
-            settings.ICOMMONS_REST_API_HOST,
-            settings.SELENIUM_CONFIG['icommons_rest_api']['base_path'],
-            os.path.join(path, ''))
-
-        if settings.ICOMMONS_REST_API_SKIP_CERT_VERIFICATION:
-            request_args['verify'] = False
-
-        return request_args, url
+            cls.search_page.select_year(year)
+        cls.search_page.submit_search(search_term)
+
+    @classmethod
+    def _load_test_course(cls):
```

Comment:
Oh, absolutely, none of the stuff I brought up should block this getting checked in.

FWIW, I'm used to seeing classmethods being defined/used primarily when they'll need to be called from the class, not from an instance.  ie. I'm used to methods defaulting to instance methods unless there's a reason they can't be (ie. factory methods, singleton classes, etc).